src/modules/Workspaces/WorkspacesSnapshotTool/SnapshotUtils.cpp

```diff
@@ -285,83 +242,16 @@ namespace SnapshotUtils
-                auto parentProcessPath = get_process_path(parentPid);
-
-                // check if original process is in the subfolder of the parent process which is a sign of an steam-like app
-                std::wstring processDir = std::filesystem::path(processPath).parent_path().c_str();
-                std::wstring parentProcessDir = std::filesystem::path(parentProcessPath).parent_path().c_str();
-
-                if (parentProcessPath == L"")
-                {
-                    if (processPath.ends_with(NonLocalizable::ApplicationFrameHost))
-                    {
-                        // filter out ApplicationFrameHost.exe
-                        continue;
-                    }
-                    else
-                    {
-                        Logger::info(L"parent process unknown, the parent app is an already closed file manager app, it is a standalone app");
-                        standaloneApp = true;
-                    }
-                }
-                else if (processDir.starts_with(parentProcessDir))
-                {
-                    Logger::info(L"parent process: {}, original process is in the subfolder of the parent process, it is a steam-like app", parentProcessPath);
-                    steamLikeApp = true;
-                }
-                else if (IsFileManagerApp(parentProcessPath))
-                {
-                    Logger::info(L"parent process: {}, The parent process is a known file manager app, it is a standalone app", parentProcessPath);
-                    standaloneApp = true;
-                }
-                else
-                {
-                    Logger::info(L"parent process: {}, The parent process is NOT a known file manager app, it is a steam-like app", parentProcessPath);
-                    steamLikeApp = true;
-                }
-
-                if (standaloneApp)
-                {
-                    data = Utils::Apps::AppData{
-                        .name = std::filesystem::path(processPath).stem(),
-                        .installPath = processPath,
-                    };
-                }
-                else if (steamLikeApp)
-                {
-                    if (!parentProcessPath.empty())
-                    {
-                        data = Utils::Apps::GetApp(parentProcessPath, installedApps);
-                        if (!data.has_value() || data->name.empty())
-                        {
-                            Logger::info(L"Installed parent app not found: {}", processPath);
-                            continue;
-                        }
-
-                        processPath = parentProcessPath;
-                    }
-                    else
-                    {
-                        Logger::info(L"Parent process path not found");
-                        continue;
-                    }
-                }
+                Logger::info(L"Installed app not found: {}", processPath);
```

Comment:
Here a 
continue; 
is needed othervise it tries to add the filtered out app (AppframeHost.exe for example) and crashes.

Reply:
Done! Thank you!